### DISCUSSION TITLE
Canned Response for New Conversation

### DISCUSSION THREAD

**Discussion by mattiamarras** [chatwoot/chatwoot#4686] [Feature Requests]
Hi, I would like to start a new conversation form the contacts list with a canned response, is it possibile? I'm trying to use the short code but nothing happens. The short code work good only in the conversations page in response to a conversation started by the customer or previously started from the contacts list?

**Comment by sojan-official** [comment]
this is available in our latest release. 
ref: https://github.com/chatwoot/chatwoot/pull/4453

**Comment by mattiamarras** [comment]
Yes, problem solved, it works on 2.5

Il giorno lun 23 mag 2022 alle ore 15:35 Sojan Jose <
***@***.***> ha scritto:
